tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -1991,6 +1991,57 @@ def test_in_semver_projects_group_resolution_stores_current_release_version(self
         for release in [release_3, release_4]:
             assert not GroupResolution.has_resolution(group=group, release=release)
 
+    def test_in_non_semver_projects_group_resolution_stores_current_release_version(self):
+        """
+        Test that ensures that when we resolve a group in the next release, then
+        GroupResolution.current_release_version is set to the most recent release associated with a
+        Group, when the project does not follow semantic versioning scheme
+        """
+        release_1 = Release.objects.create(
+            organization_id=self.project.organization_id, version="foobar 1"
+        )
+        release_1.update(date_added=timezone.now() - timedelta(minutes=45))
+        release_2 = Release.objects.create(
+            organization_id=self.project.organization_id, version="foobar 2"
+        )
```

Comment:
jfyi, you can use `self.create_release(version='foobar 1', date_added=timezone.now() - timedelta(minutes=45))`. This automatically creates a release related to `self.organization` and `self.project`

Reply:
Nice x2